client/src/app/contact/contactTmpl.html

```diff
@@ -1,7 +1,7 @@
 <md-content id="template-contact" ng-controller="contactCtrl">
     <form name="formContact" ng-submit="send()" class="text-center" novalidate>
         <p><strong>Questions or comments? Leave us a message!</strong></p>
-        <fieldset ng-disabled="submitted">
+        <fieldset ng-disabled="submitting || submitted">
```

Comment:
Wondering if we should XOR this cause technically you can't be submitted if it's submitting and vice versa.